### DISCUSSION TITLE
Understanding shapesys modifier

### DISCUSSION THREAD

**Discussion by andrewfowlie** [scikit-hep/pyhf#2548] [Q&A]
### Summary

I’m trying to understand the shapesys modifer. I’ve read the documentation at https://pyhf.readthedocs.io/en/v0.7.6/intro.html#tab-modifiers-and-constraints

I understand that this modifier contributes a factor to the likelihood 

Poisson( naux | gamma naux)

where gamma is the multiplicative modifier and 

naux = (b / delta b)^2. 

That is, it’s the pmf for a Poisson for naux counts given gamma naux expected counts. 

I don’t understand this as naux isn’t in general an integer. 

Please can you clarify:

- what is the interpretation of this term as a likelihood function? E.g., do we drop the factorial term in the Poisson pmf, replace it with a gamma function or something else?
- what is the interpretation of this term as a piece of a statistical model? I.e., does this represent a sampling distribution for naux? If so how since the Poisson distribution is a discrete distribution and naux isn’t an integer? Does it represent a prior for the parameter gamma? If so how since it isn’t a density?
- more generally, what motivates this choice? 

Many thanks 🙏 

### Additional Information

```python (paste below)

```

### Code of Conduct

- [X] I agree to follow the Code of Conduct

**Comment by matthewfeickert** [comment]
@andrewfowlie Thanks for the question (which I've migrated to a GitHub Discussion). The developer team is mostly traveling for work at the moment but we'll get back to you soon.

**Reply by andrewfowlie** [reply]
Any clarification would be appreciated. I’m trying to interpret pyhf spec as a statistical model
